src/main/java/org/iac2/common/model/compliancerule/ComplianceRule.java

```diff
@@ -10,16 +10,19 @@
 import org.iac2.common.model.compliancerule.parameter.ComplianceRuleParameter;
 import org.iac2.common.model.compliancerule.parameter.DoubleComplianceRuleParameter;
 import org.iac2.common.model.compliancerule.parameter.IntegerComplianceRuleParameter;
+import org.iac2.common.model.compliancerule.parameter.StringCollectionComplianceRuleParameter;
 import org.iac2.common.model.compliancerule.parameter.StringComplianceRuleParameter;
 
 @Setter
 @Getter
 @NoArgsConstructor
 public class ComplianceRule {
+    private Long id;
     private String type;
     private String location;
     private Collection<ComplianceRuleParameter> parameterAssignments;
-    public ComplianceRule(String type, String location) {
+    public ComplianceRule(Long id, String type, String location) {
+        this.id = id;
```

Comment:
yes, it is the same id generated in the database. We need it because we need to uniquely identify compliance rules that we move around between the checking plugin and the fixing plugin...